cmd/token.go

```diff
@@ -24,11 +26,18 @@ func init() {
 
 	loginCmd.Flags().BoolVarP(&isUserToken, "user-token", "u", false, "Whether to login as a user or getting an app access token.")
 	loginCmd.Flags().StringVarP(&userScopes, "scopes", "s", "", "Space seperated list of scopes to request with your user token.")
+	loginCmd.Flags().StringVarP(&revokeToken, "revoke", "r", "", "Instead of generating a new token, revoke the one passed to this parameter.")
+	loginCmd.Flags().StringVarP(&overrideClientId, "client-id", "i", "", "Override/manually set client ID for token actions. By default client ID from CLI config will be used.")
 }
 
 func loginCmdRun(cmd *cobra.Command, args []string) {
 	clientID = viper.GetString("clientId")
 	clientSecret = viper.GetString("clientSecret")
+
+	if overrideClientId != "" {
+		clientID = overrideClientId
+	}
+
```

Comment:
This will have the potential of being triggered due to line 44 as well- if the user had never set up the CLI previously, it'll use the configured option. 

You can move this below that or add another conditional to skip that if using a revoke.

Reply:
Moving sounds good, will do!